axum/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
+- **fixed:** `QueryRejection` response exposes type names through public interface ([#1171])
```

Comment:
This almost sounds like the fix was to expose internal types. Maybe this is more clear?

```suggestion
- **fixed:** Don't expose internal type names in `QueryRejection` responses ([#1171])
```

Reply:
huh.. I guess you are right.